### PR TITLE
updates to zombie processing

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

File: logic/zombie.go [hunk 1]
```diff
@@ -39,7 +37,36 @@ func CheckZombies(newnode *models.Node, mac net.HardwareAddr) {
 		}
 		if node.HostID == newnode.HostID || time.Now().After(node.ExpirationDateTime) {
 			logger.Log(0, "adding ", node.ID.String(), " to zombie list")
-			newZombie <- node.ID
+			zombies = append(zombies, node.ID)
```
Comment (0xdcarns) [inline logic/zombie.go:40]: may need to synchronize these writes

Comment (0xdcarns) [reply logic/zombie.go:40]: to both slices

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Review by theguy951357 — APPROVED** [review]
works pending changes

**Review by 0xdcarns — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
testing steps done in GRA-1085
pass

**Pull request merged**